app/oapv_app_enc.c

```diff
@@ -856,9 +856,17 @@ int main(int argc, const char **argv)
                     }
                 }
 
+                unsigned char bs_size[4];
+                oapv_bsw_write_direct(bs_size, stat.write, 32);
```

Comment:
this function cannot be called from application code.
Please let me know what you want first.